backend/controllers/project.controller.js

```diff
@@ -67,4 +67,26 @@ ProjectController.destroy = async function (req, res) {
 };
 
 
+ProjectController.getProjectManagers = async function (req, res) {
+  try {
+    const userProjectMap = {};
+    const projects = await Project.find({
+      managedByUsers: { $exists: true, $ne: [] }
+    });
+
+    for (const project of projects) {
+      for (const user of project.managedByUsers) {
+        if (!userProjectMap[user]) {
+          userProjectMap[user] = [];
+        }
+        userProjectMap[user].push(project.name);
```

Comment:
Ref: #1771 
It seems like you might have missed the capture of the `project.id`. Perhaps pushing an project object with `id` and `name` properties would be great. Other than that, the code block looks good!

Reply:
This is a good catch